.cicd/platforms/ubuntu20.Dockerfile

```diff
@@ -6,11 +6,13 @@ RUN apt-get update && apt-get upgrade -y && \
                        cmake                \
                        git                  \
                        jq                   \
-                       libboost-all-dev     \
                        libcurl4-openssl-dev \
                        libgmp-dev           \
                        libssl-dev           \
                        llvm-11-dev          \
                        ninja-build          \
+                       python3-all          \
```

Comment:
We should be "short and sweet" with the dependency list we ask users to install. Is `python3-all` really required to build & test leap? Having ` python3-numpy` as a requirement already gets us `python3`.

Reply:
Without it I had the following issue in `Build leap-dev package (ubuntu 20)`:

```
Preparing to unpack .../leap-dev_4.1.0-dev-ubuntu20.04_amd64.deb ...
Unpacking leap-dev (4.1.0-dev) ...
Setting up leap (4.1.0-dev) ...
Setting up leap-dev (4.1.0-dev) ...
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: cannot import name 'sysconfig' from 'distutils' (/usr/lib/python3.8/distutils/__init__.py)
dpkg: error processing package leap-dev (--configure):
 installed leap-dev package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 leap-dev
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Reply:
let me try with adding only `python3-distutils`

Reply:
right but that error occurs when _installing_ `leap-dev.deb` -- not when building or running the tests.

Reply:
So what do you suggest instead?

Reply:
Seems like using `python3-distutils` was enough (better than `python3-all ` for sure)

Reply:
I don't think you need `python3-distutils` here -- it only needs to be in the `leap-dev.deb` dependencies, because only there is `distutils` used.

Reply:
I tried with #b2b0af0 and it didn't work, so reverting.

Reply:
Yeah that was a failure unrelated to your change -- #1437

Reply:
With #1438 now merged to main, getting that merged here will resolve that problem

Reply:
Sure, I'll merge this.